src/lib/wallet/utils.js

```diff
@@ -25,9 +25,9 @@ export const weiToGd = (wei: number): number => (wei * Math.pow(0.1, DECIMALS)).
 /**
  * convert gooddollars to wei (0 decimals) use toFixed to overcome javascript precision issues ie 8.95*Math.pow(0.1,2)=8.9500000001
  * @param {string} gd
- * @returns {number}
+ * @returns {string}
  */
-export const gdToWei = (gd: number): number => (gd * Math.pow(10, DECIMALS)).toFixed(0)
+export const gdToWei = (gd: string): string => (gd * Math.pow(10, DECIMALS)).toFixed(0)
```

Comment:
why string?

Reply:
Because the input is a text type input, so the parameter we have is a string...anyway it works fine...and it returns a string because we use `toFixed()`, that convert the number to string